src/ws-tiles-chart/ws-tiles-chart.js

```diff
@@ -0,0 +1,93 @@
+import {Component, PropTypes} from "../imports";
+import {Tile} from "./tile";
+
+/**
+ * This class describes a Preact/React component which renders a tiles chart
+ * @class WSTilesChart
+ * @property {Object} props React properties object
```

Comment:
The possible properties should be documented here. Pls check other components like ws-dropdown